### DISCUSSION TITLE
Add `pdk validate plans`

### DISCUSSION THREAD

**Discussion by sanfrancrisko** [puppetlabs-toy-chest/pct#193] [Ideas]
Originally raised in [PDK-1316](https://tickets.puppetlabs.com/browse/PDK-1316) by @scotje :

Currently `pdk validate` skips plan manifests when validating puppet code since validating them requires additional params to be passed to Puppet. We should add a specific validator for plan manifests and include it in the default `pdk validate` invocation.

**Comment by sanfrancrisko** [comment]
See also: https://github.com/puppetlabs/bolt/issues/1699

**Comment by davidsandilands** [comment]
Just to tag this has been an issue for some PE customers @Adam-Shambrook

**Comment by jay7x** [comment]
See https://github.com/puppetlabs/pdk/pull/1142. We can revive and merge it.
